README.md

```diff
@@ -24,16 +24,21 @@ You will need the following:
 
 ### What's Working? ###
 Most features of the game are available -- and we're at a state where you can complete the full campaign without issue.
+##### Original Features #####
 - Single player modes (Campaigns, custom levels, custom campaigns)
```

Comment:
The original game did not support custom levels or custom campaigns, those are BIG features of CorsixTH